liam2/data.py

```diff
@@ -710,12 +711,15 @@ def load_path_globals(globals_def):
     localdir = config.input_directory
     globals_data = {}
     for name, global_def in globals_def.iteritems():
-        if 'path' not in global_def:
+        if 'path' not in global_def and 'value' not in global_def:

```

Comment:
"constant" globals shouldn't be handled in load_path_globals. Either in index_tables directly or in a separate method "handle_constant_globals".